grisp/grisp_base/sys/erl_main.c

```diff
@@ -75,7 +75,7 @@ static int argc;
 static char *strdupcat (char *s1, char *s2)
 {
   char *res;
-  
+
```

Comment:
Sorry to nitpick, but can you remove this change? It would make it easier for us to bisect in the future if no changes to extra files were added

Reply:
Sure, sorry ! I've already spotted that one in the other PR I should have fixed it before.